client/client.py

```diff
@@ -0,0 +1,47 @@
+from pathlib import Path
+
+import requests
+
+from file_data_transfer_api.api_datamodel import FileDatabaseEntry
+
+class Client:
+    def __init__(self):
+        self.url = "http://127.0.0.1:8000"
+
+    def upload_file(self, path_to_file: Path) -> FileDatabaseEntry:
+        file_to_upload = {'file': open(path_to_file, 'rb')}
+        response = requests.post(url=f"{self.url}/files",files=file_to_upload)
+        if response.ok:
+            return FileDatabaseEntry(**response.json())
+        else:
+            raise ValueError
+
+    def download_file(self, file_id: str, path_to_save: Path) -> None: # clients act as an interface to interact with apis
```

Comment:
`-> None` should probably be omitted 

```suggestion
    def download_file(self, file_id: str, path_to_save: Path): # clients act as an interface to interact with apis
```